Sources/TSCBasic/FileSystem.swift

```diff
@@ -491,8 +491,8 @@ private struct LocalFileSystem: FileSystem {
 
     func readFileContents(_ path: AbsolutePath) throws -> ByteString {
         // Open the file.
-        let fp = fopen(path.pathString, "rb")
-        if fp == nil {
+        let fpo = fopen(path.pathString, "rb")
+        guard let fp = fpo else {
```

Comment:
Why not merge these two lines?

```suggestion
        guard let fp = fopen(path.pathString, "rb") else {
```

Reply:
Good point, will change.

Reply:
Done.